text/src/autogluon/text/automm/optimization/utils.py

```diff
@@ -480,3 +488,30 @@ def apply_layerwise_lr_decay(
         parameter_group_names[group_name]["params"].append(name)
 
     return list(parameter_group_vars.values())
+
+
+def config_update_loss_func(
```

Comment:
Could we change `config_update_loss_func` to `update_config_by_rules`? I guess we will have other similar needs to update config based on our pre-defined rules. So, we can make this function name more general and add other similar stuff into this function in future.